yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -89,6 +90,13 @@ export { deployAndInitializeTokenAndBridgeContracts } from '../shared/cross_chai
 
 const { PXE_URL = '' } = process.env;
 
+const telemetry = createAndStartTelemetryClient(getTelemetryConfig(), 'aztec-test');
```

Comment:
Where is the config for the e2e tests defined? As I understand, this is going to use the no-op client if collector is not defined, but then where do the stats go?

Reply:
I'll make it clearer (although if we switch to a global instance it might not be needed).